libs/components/core/src/lib/modules/media-query/media-breakpoint-type.ts

```diff
@@ -0,0 +1,65 @@
+import { SkyMediaBreakpoints } from './media-breakpoints';
+
+/**
+ * @internal
+ */
+export const SKY_MEDIA_BREAKPOINT_TYPES = ['xs', 'sm', 'md', 'lg'] as const;
+
+/**
+ * The names for all media breakpoints.
+ */
+export type SkyMediaBreakpointType =
+  (typeof SKY_MEDIA_BREAKPOINT_TYPES)[number];
+
+/**
+ * Whether the value is of type `SkyMediaBreakpointType`.
+ */
+export function isSkyMediaBreakpointType(
+  value: SkyMediaBreakpointType | SkyMediaBreakpoints | null | undefined,
+): value is SkyMediaBreakpointType {
+  return (
+    value !== null &&
+    value !== undefined &&
+    SKY_MEDIA_BREAKPOINT_TYPES.includes(value as SkyMediaBreakpointType)
+  );
+}
+/**
+ * The default media breakpoint type.
+ * @internal
+ */
+export const SKY_MEDIA_BREAKPOINT_TYPE_DEFAULT: SkyMediaBreakpointType = 'md';
```

Comment:
The concept of a default breakpoint is odd to me. When would it be set to a default? When there is no screen? If it's for testing, I'd prefer the consumer have to explicitly set a breakpoint before making any assertions about the screen size.

Reply:
Good call. I converted `breakpointChange` to a ReplaySubject (instead of a BehaviorSubject) so it won't require a default value (I made the same change to the testing controller).